src/views/Application.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="container py-4">
+    <div class="row">
+      <div class="col">
+        <a-card hoverable style="width: 100%">
+          <template class="ant-card-actions" slot="actions">
+            <a-icon type="api" title="Instalar agora" @click="installApp" v-if="!installed"/>
+            <a-icon type="setting" title="Configurar aplicativo" @click="tongleSettings" v-if="installed"/>
```

Comment:
`tongle`? seria `toggle`?
```suggestion
            <a-icon type="setting" title="Configurar aplicativo" @click="toggleSettings" v-if="installed"/>
```

Reply:
sim :$